README.md

```diff
@@ -1,2 +1,110 @@
+
+## AWS Services
+
+For this example we will multiple AWS services, such as [AWS S3](https://aws.amazon.com/s3/) and [Amazon Transcribe](https://aws.amazon.com/transcribe/).
+
+In S3, create a bucket that will be used to store the MP3 files retrieved from Nexmo. Then make note of the bucket name for later inclusion to the `.env` file.
+
+Using the [AWS IAM](https://aws.amazon.com/iam/), add a new user and add the following permissions by importing the following JSON.
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": "transcribe:*",
+            "Resource": "*"
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": [
+            "s3:PutObject",
+            "s3:GetObject",
+            "s3:DeleteObject"
+        ],
+            "Resource": "arn:aws:s3:::{bucket_name}/*"
+        }
+    ]
+}
+```
+
+
+## Running the App
+
+This sample app uses a `.env` file to provide the API key and URL to the IBM Tone Analyzer API.
+
+Copy the provided `.env.example` file to a new file called `.env`:
+
+```
+cp .env.example > .env
+```
+
+Then update the values with those from the IBM Tone Analyzer service Manage page, and then save.
+
+```
+APP_ID=voice-aws-transcribe-php
+LANG_CODE=en-US
+SAMPLE_RATE=8000
+AWS_REGION=us-east-1
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_S3_BUCKET_NAME=
+AWS_S3_RECORDING_FOLDER_NAME=
+NEXMO_APPLICATION_PRIVATE_KEY_PATH=
+NEXMO_APPLICATION_ID=
+```
+
+### Use Composer to install dependencies
+
+Both methods of serving the app, shown below, do require the use of Composer to install dependencies and set up the autoloader.
+
+Assuming a Composer global installation. [https://getcomposer.org/doc/00-intro.md#globally](https://getcomposer.org/doc/00-intro.md#globally)
+
+```
+composer install
+```
+
+#### Launching the PHP built-in webserver
+
+We will utilize the PHP built-in webserver for this sample application. Though it not intended for production, it is fine for sample/test usage. To serve the app run the following command in a terminal from the application directory:
+
+```
+php -S localhost:8080
+```
+
+## Launch ngrok
+To enable the Nexmo callback to reach the application, expose the application to the internet using [ngrok](https://ngrok.com/). To see how, [check out this guide](https://www.nexmo.com/blog/2017/07/04/local-development-nexmo-ngrok-tunnel-dr/).
+
+## Linking the app to Nexmo
+
+After buying a virtual phone number through the [Nexmo Dashboard](https://dashboard.nexmo.com/buy-numbers), create an application with Voice capabilities and link it to the number. Make sure to generate the public/private keys, and save the private key in the application root when prompted.
+
+As part of the Application creation, we also need to add the URLs for Event and Answer callbacks, while Fallback is not needed for this example.
```

Comment:
Should we link to the dashboard's applications page here, or provide examples using the Nexmo CLI?

Reply:
I had that originally but opted for the dashboard because I think many new users may feel more comfortable going that route.